packages/repo-metadata-lint/src/utils/octokit-util.ts

```diff
@@ -0,0 +1,183 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Octokit} from '@octokit/rest';
+// Conflicting linters think the next line is extraneous or necessary.
+// eslint-disable-next-line node/no-extraneous-import
+import {ProbotOctokit} from 'probot';
+import {promisify} from 'util';
+import {readFile} from 'fs';
+import {request} from 'gaxios';
+import {sign} from 'jsonwebtoken';
+
+const readFileAsync = promisify(readFile);
```

Comment:
12 and up, we can use `fs.promises` right?

Reply:
fixed.